src/Blazored.Toast/BlazoredToast.razor.css

```diff
@@ -104,7 +104,7 @@
     height: 6px;
     border-bottom-left-radius: .375rem;
     background-image: linear-gradient(rgba(0, 0, 0, 0.1) 0 0);
-    transition: all .5s linear;
+    transition: all .1s linear;
```

Comment:
Why was this change made?

Reply:
I didn't know how to solve the issue otherwise when you move mouse on the toast progress bar will actually be stopped after 0.5 sec so it's not a good user experience, it's not reacting immediately and i guess its because of this 0.5 sec animation in CSS

with 0.5 sec

https://user-images.githubusercontent.com/506853/219736216-45d27894-0cd1-40f5-8956-32c511ab444c.mp4


with 0.1 sec

https://user-images.githubusercontent.com/506853/219736382-610736cd-cb05-4749-b7a3-4986831c6e91.mp4


but at the end I changed it to hide the progress bar so it shouldn't matter actually. (still it will be 0.5 sec more progress bar taken when user move mouse out)

https://user-images.githubusercontent.com/506853/219736792-83a9f0c4-fc3f-4743-afdc-aa65c8712c67.mp4

Reply:
Yep, makes sense.